app/frontend/src/features/communities/events/LongEventCard.tsx

```diff
@@ -0,0 +1,203 @@
+import {
+  Card,
+  CircularProgress,
+  Typography,
+  useMediaQuery,
+  useTheme,
+} from "@material-ui/core";
+import Avatar from "components/Avatar";
+import { TO } from "features/constants";
+import { Event } from "proto/events_pb";
+import { useMemo } from "react";
+import { Link } from "react-router-dom";
+import { routeToEvent } from "routes";
+import { timestamp2Date } from "utils/date";
+import dayjs from "utils/dayjs";
+import makeStyles from "utils/makeStyles";
+
+import { getAttendeesCount, ONLINE } from "../constants";
+import getContentSummary from "../getContentSummary";
+import { getExtraAvatarCountText } from "./constants";
+import eventImagePlaceholder from "./eventImagePlaceholder.svg";
+import { useEventAttendees } from "./hooks";
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
Oh crap, that would have been much easier to implement as well 🙃  I'll change it since I don't really like the avatar thing at the preview level tbh, seems to fetching a lot of data for very little gain